scripts/apps/archive/services/ArchiveService.ts

```diff
@@ -125,9 +126,7 @@ export function ArchiveService(desks, session, api, $q, search, $location, confi
      * @param {Object} item
      * @return boolean if the state of the item is in one of the published states, false otherwise.
      */
-    this.isPublished = function(item) {
-        return _.includes(['published', 'killed', 'scheduled', 'corrected', 'recalled'], item.state);
-    };
+    this.isPublished = isPublished;
```

Comment:
Can you remove it from here and `AuthoringService` and import it directly where needed? It's hard to track usages when the same thing is aliased in multiple places.